### PR TITLE
Clients ali

### PR DESCRIPTION
**Pull request by AliRezaHassanloo2731**
Add Clients.js and clientList.js and Logo.js

### CODE REVIEW

File: frontend/app/_components/ClientList.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import Client from "./Client";
+
+const clients = [
```
Comment (magdazelena) [inline frontend/app/_components/ClientList.js:3]: I assume this is for testing purposes but as a rule you should not have these commited as part of production code. If you choose to do so, a good practice is to either have mock data in json files or at least name the variable mockdata or something:)

File: frontend/app/_components/ClientList.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+];
+
+function ClientList() {
+  return (
+    <ul className="flex flex-col gap-1  mb-8 text-lg">
+      {clients.map((client, idx) => (
+        <Client
+          client={client}
+          // onSelectFriend={}
+          // selectedFriend={}
```
Comment (magdazelena) [inline frontend/app/_components/ClientList.js:31]: it's good not to leave out commented code and only implement the functionalities you need in the particular moment/for particular PR

File: frontend/app/layout.js [hunk 3]
```diff
@@ -1,27 +1,32 @@
...
-import { SessionProvider } from 'next-auth/react';
+import localFont from "next/font/local";
+import "./_styles/globals.css";
+import { SessionProvider } from "next-auth/react";
 
 const geistSans = localFont({
-  src: './fonts/GeistVF.woff',
-  variable: '--font-geist-sans',
-  weight: '100 900',
+  src: "./fonts/GeistVF.woff",
```
Comment (magdazelena) [inline frontend/app/layout.js:6]: this seems like changes I already seen in some other PR of yours. Please make sure that your PRs only contain the new code that changes. Take a look into rebasing functionality of git for that :)

**Review by magdazelena — CHANGES_REQUESTED** [review]
some small stuff :)

**Review by KristofferTolboll2 — COMMENTED** [review]
I agree with @magdazelena. 

Furthermore I want to add that your `app/_components` file contains a mix of components like (`Logo` and `Client`) as well as pages like `SignIn` and `SignUp`. 

THis will become confusing in larger projects. 

There are multiple correct way to do this. One way is to use a pattern like this: 

```
src/
  _components/
    Header.js
    Footer.js
  pages/
    index.js
``` 


Consider what kind of setup you guys want, and let's discuss it next time.

File: frontend/app/_components/ClientList.js (hunk 1 above)
Comment (AliRezaHassanloo2731) [reply frontend/app/_components/ClientList.js:3]: Thank you for the feedback. Next time, we'll ensure the data is sourced directly from our database, as intended. You're absolutely right about the naming convention—I'll make sure to use descriptive names like mockData or similar in the future. I appreciate your guidance as I work to improve these practices.

File: frontend/app/_components/ClientList.js (hunk 2 above)
Comment (AliRezaHassanloo2731) [reply frontend/app/_components/ClientList.js:31]: yes, exactly

File: frontend/app/layout.js (hunk 3 above)
Comment (AliRezaHassanloo2731) [reply frontend/app/layout.js:6]: this is not my code, I just change meta data

**Pull request merged**